CardPayments/src/main/java/com/paypal/android/cardpayments/CardResult.kt

```diff
@@ -1,19 +1,21 @@
 package com.paypal.android.cardpayments
 
-import android.net.Uri
-
 /**
  * A result returned by [CardClient] when an order was successfully approved with a [Card].
  *
  * @property [orderId] associated order ID.
  * @property [liabilityShift] Liability shift value returned from 3DS verification
+ * @property [status] status of the order
+ * @property [didAttemptThreeDSecureAuthentication] 3DS verification was attempted.
+ * Use v2/checkout/orders/{orderId} in your server to get verification results.
  */
 data class CardResult(
     val orderId: String,
 
-    /**
-     * @suppress
-     */
-    val deepLinkUrl: Uri? = null,
```

Comment:
This is technically breaking too right - can we deprecate this instead?

Reply:
We could yeah.

Reply:
Updated: [933fe41](https://github.com/paypal/paypal-android/pull/266/commits/933fe41b6a67716209918f5bd870fb949ddd61f5)